docs/development/accessibility.md

```diff
@@ -0,0 +1,120 @@
+# Accessibility
+
+In the context of software development accessibility refers to
+the ability of people with disabilities to effectively
+use the software being developed. When designing interactions
+through the screen, keyboard, mouse and speakers it is
+important to ensure that people with disabilities will be
+able to effectively participate in those interactions.
+
+For software developed in Node.js the user interface (UI) typically
+falls into one of the following:
+
+1) Browswer based UI delivered though JavaScript, HTML and CSS.
+1) Desktop type application built on something like
+   [electron](https://www.electronjs.org/).
+1) Command line scripts/applications.
+
+Both 1) and 2) above leverage JavaScript,
+HTML and CSS in a similar manner and we'll use
+web based applications to describe both of these.
+
+In web based applications, the HTML, JavaScript and CSS run
+in the browser (or equivalent in frameworks like electron)
+and is referred to as the front-end. The back-end running
+on Node.js provides the business logic to the front
+end through REST APIs, websockets or some other set of APIs.
+
+In this context, the focus on accessibility is in the
+development of the front-end as opposed to
+JavaScript running under Node.js in the back-end.
+
```

Comment:
```suggestion

## Legal Requirements
We have outlined above the moral obligation to ensure that our software is accessible. Many jurisdictions also mandate software accessibility by law. For example, in [Israel](https://www.gov.il/he/Departments/Guides/website_accessibility?chapterIndex=3), websites that offer a public service (to exclude, e.g. a private blog) must conform to WCAG 2.0 at the AA level, and businesses that do not comply may be liable for statutory damages.
```

This may need to include legal disclaimers, IANAL, etc.